src/__snapshots__/extensions.spec.ts.snap

```diff
@@ -2457,9 +2457,9 @@ exports[`Markdown extensions for interpolation works inside explicit markdown ta
                   ],
                   "position": {
                     "end": {
-                      "column": 27,
+                      "column": 26,
```

Comment:
What causes the difference in token position?

Reply:
I didn't investigate.. we don't make use of the position information anyway.. :shrug: